src/components/Candidates/Candidate.vue

```diff
@@ -1,21 +1,26 @@
 <script setup>
 import EditButton from "../common/EditButton.vue";
 import DeleteButton from "../common/DeleteButton.vue";
-
-let singleCandidate = {
-  position: "position 1",
-  username: "username 1",
-  linkedinUrl: "linkedinUrl 1",
-  feedback: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
-    sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
-    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip 
-    ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse 
-    cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, 
-    sunt in culpa qui officia deserunt mollit anim id est laborum.`,
-  avatarUrl:
-    "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/682.jpg",
-  id: "1",
-};
+import { useRoute } from "vue-router";
+import { onMounted, ref } from "vue";
+import { useCandidateStore } from "../../stores/candidates";
+const { params } = useRoute();
+const currentCandidate = ref({});
+onMounted(async () => {
+  try {
+    const { getCandidateById } = useCandidateStore();
+    const { data } = await getCandidateById(params.id);
+    if (data.length > 1) {
+      currentCandidate.value = data.filter((candidate) => {
```

Comment:
Why you use `filter`?
`:GET https://63510b67dfe45bbd55b4fc4d.mockapi.io/candidates/1`
In Results:
```
{
  "position": "position 1",
  "username": "username 1",
  "linkedinUrl": "linkedinUrl 1",
  "feedback": "feedback 1",
  "avatarUrl": "https://cloudflare-ipfs.com/ipfs/Qmd3W5DuhgHirLHGVixi6V76LhCkZUz6pnFt5AJBiyvHye/avatar/602.jpg",
  "id": "1"
}
```

Reply:
because mockApi returns 10 objects if you call this request :GET https://63510b67dfe45bbd55b4fc4d.mockapi.io/candidates/1

Reply:
auth0 fixed

Reply:
> because mockApi returns 10 objects if you call this request :GET https://63510b67dfe45bbd55b4fc4d.mockapi.io/candidates/1
![Screenshot 2022-11-17 at 12 56 18](https://user-images.githubusercontent.com/10779650/202415376-92f88ffe-72f3-47f2-a664-ed65f43549b1.jpg)